draft-dekater-scion-dataplane.md

```diff
@@ -447,8 +449,18 @@ The SCION common header has the following packet format:
 | other          |      |                      | Unassigned     |
 {: #table-3 title="Allocations of type values to length values"}
 
-- `RSV`: These bits are currently reserved for future use.
+A service address designates a set of endpoint addresses rather than a singular one. A packet addressed to a service is redirected to any one endpoint-addresses that is known to be part of the set.
```

Comment:
Based on table 3, It seems that an address is a service address if either `DT/DL`=1/0 or if `ST/SL`=1/0. Is source ever used? If not, it should be clearer. Maybe we could just say, before the service address table, that those addresses are for case  `DT/DL`=1/0

Reply:
Made it more obvious that this goes for both src and dst.